File: Code/lecture6/pset6/sentimental-mario-less/mario.py
from cs50 import get_int

# get height of pyramid form user
while True:
    height = get_int("Height: ")
    if (height > 0 and height <= 8):
        break

# print the pyramid
for i in range(height):

    print(" " * (height-(i+1)), end="")
    print("#" * (i+1))

File: Code/lecture1/pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int h;

    //get user input
    do
    {
        h = get_int("Height: ");
    }
    while (h > 8 || h < 1);

    //loop for height
    for (int i = 0; i < h; i++)
    {
        //loop for left alignment
        for (int k = h - 1; k > i; k--)
        {
            printf(" ");
        }
        //loop for number of # per row
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        //move to next row after completing the row
        printf("\n");
    }


}